lib/services/audio/mobile_audio_player.dart

```diff
@@ -312,8 +315,13 @@ class MobileAudioPlayer {
   }
 
   Future<void> seekTo(Duration position) async {
+    _durationTimer.cancel();
     await _setBufferingState();
     await _audioPlayer.seek(position);
+    await _persistState(LastState.playing, _audioPlayer.position)
+        .whenComplete(() => _durationTimer = Timer.periodic(Duration(seconds: 10), (timer) async {
```

Comment:
Why canceling and then creating another timer here? If it is playing the timer should already be running.

Reply:
Done